godot_project/editor/input_handlers/molecular_structures/transform_input_handler.gd

```diff
@@ -431,6 +439,8 @@ func _force_gizmo_update() -> void:
 		_selection_initial_position = _init_initial_positions_and_determine_center()
 		_helper.global_transform.basis = Basis()
 		_helper.global_position = _selection_initial_position
+	if _other_handler_has_exclusivity:
+		GizmoRoot.disable_gizmo()
```

Comment:
The transform gizmo is force shown when the selection changes, which is an issue because the auto posing feature changes the selection when creating new atoms. 
This part lets the gizmo update, but hides it if another handler has exclusivity.